install.sh

```diff
@@ -232,7 +232,7 @@ echo "REACT_APP_NGINX_PORT=$nginxport" >> .env
 echo "REACT_APP_NGINX_IP=$nginxip" >> .env
 echo "REACT_SERVER_IP=$reactclientip" >> .env
 echo "REACT_SERVER_PORT=$reactclientport" >> .env
-echo "BASIC_AUTH_PASS=$basicpass" >> .env
+echo "ADMIN_AUTH_PASS=$basicpass" >> .env
```

Comment:
Let's still keep `BASIC_AUTH_PASS` and add `ADMIN_AUTH_PASS`

Reply:
to fully deprecate `BASIC_AUTH` we need to remove the nginx docker definition and all reference to it